draft-polli-ratelimit-headers.md

```diff
@@ -937,16 +910,16 @@ At this point you should stop increasing your request rate.
 
    I'm open to suggestions. File an issue if you think it's worth ;).
 
-5. Why using delta-seconds instead of UNIX Timestamp? Why IMF-fixdate is NOT RECOMMENDED?
+5. Why using delta-seconds instead of a UNIX Timestamp?
    Why not using subsecond precision?
 
-   Using delta-seconds permits to align with `Retry-After`, which is returned in similar contexts,
+   Using delta-seconds uniforms with `Retry-After`, which is returned in similar contexts,
```

Comment:
"uniforms" primarily means a type of clothing. Using just the word "aligns" would be clearer here.